crates/bevy_render/src/color.rs

```diff
@@ -10,7 +11,7 @@ use bevy_property::Property;
 use serde::{Deserialize, Serialize};
 use std::ops::{Add, AddAssign, Mul, MulAssign};
 
-/// A RGBA color
+/// RGBA color in linear colorspace and 32-bit per component.
```

Comment:
'linear colorspace' is underspecified. I'd recommend
```suggestion
/// RGBA color in the Linear sRGB colorspace (often colloquially referred to as "linear", "RGB", or "linear RGB").
```